script.js

```diff
@@ -0,0 +1,35 @@
+// Requirements:
+// 1. user should click on different elements to create a move
+// 2. every click alternates between x and o
+// 3. when a cell is marked, display a corresponding color
+// 4. marked cell can't be undone
+// 5. add reset button will clear contents
+
+// Outline:
+// 1. listen for a click event on a box
+// 2. when click on a box is executed, toggle between marking x and o
+
+
+var playerMoves = [];
+var storePlayerMoves = playerMoves.push();
+var getMoves = document.getElementsByClassName('square');
+
+for (var i=0; i<playerMoves.length; i++) {
+  square[i].addEventListener('click');
```

Comment:
So when we addEventListeners to something. It needs to be a DOM element. Where is the variable square being defined in your code base? I think you would need to do something like this above the loop:

```
var squareEls = document.getElementsbyClassName("square")
```

then inside your loop you can access each of the DOM elements with class square. Additionally, addEventListener requires 2 arguments. The first is what the element should listen for(ie. `click` which you included) the second argument is the callback( the thing you want to happen when the event is triggered) I think in your case you wanted the callback to be `toggleButtonDesign`